package.json

```diff
@@ -36,7 +36,7 @@
   },
   "devDependencies": {
     "@octokit/rest": "^16.26.0",
-    "@ts-common/azure-js-dev-tools": "^0.7.0",
+    "@ts-common/azure-js-dev-tools": "^22.4.21",
```

Comment:
wow that's a BIG version bump!

Reply:
yeah and it breaks the gulpfile.ts script though 😄